### PR TITLE
feat(docs): add copy feat for demo

### PR DESCRIPTION
**Pull request by Ryan2128**
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: website/i18n/component.json [hunk 1]
```diff
@@ -35,7 +38,10 @@
     "demo-block": {
       "hide-text": "Hide",
       "show-text": "Expand",
-      "button-text": "Try it!",
+      "copy-button-text": "Copy",
+      "copy-success": "Copy successfully！",
```
Comment (JeremyWuuuuu) [inline website/i18n/component.json:42]: "Copied"

File: website/i18n/component.json [hunk 2]
```diff
@@ -4,7 +4,10 @@
     "demo-block": {
       "hide-text": "隐藏代码",
       "show-text": "显示代码",
-      "button-text": "在线运行",
+      "copy-button-text": "复制",
+      "copy-success": "复制成功！",
```
Comment (JeremyWuuuuu) [inline website/i18n/component.json:8]: “已复制”

File: website/i18n/component.json [hunk 3]
```diff
@@ -4,7 +4,10 @@
     "demo-block": {
       "hide-text": "隐藏代码",
       "show-text": "显示代码",
-      "button-text": "在线运行",
+      "copy-button-text": "复制",
```
Comment (JeremyWuuuuu) [inline website/i18n/component.json:7]: “复制代码片段”

Comment (JeremyWuuuuu) [reply website/i18n/component.json:7]: Note: no need to change English translation with this, simply remain "copy"

**Review by zazzaz — APPROVED** [review]

**Pull request merged**